src/universal-dapp.js

```diff
@@ -598,10 +618,19 @@ UniversalDApp.prototype.getCallButton = function (args) {
     })
   }
 
+  var title
+  if (isConstructor) {
+    title = 'Create'
```

Comment:
That is only shown on the top for `Create` and never mixed with actual functions. Note that the behaviour of that is unchanged.